anchor_test.py

```diff
@@ -1,19 +1,20 @@
+from typing import Sequence, Union, Set
 
-from keebgen.geometry_base import Anchors, CuboidAnchors
+from keebgen.geometry_base import AnchorCollection, CuboidAnchorCollection, LabeledPoint
 from keebgen.connector import Connector
 
 cube_input_corners = [
-            ((0, 0, 0), ('bottom', 'left', 'back')),
-            ((1, 0, 0), ('bottom', 'right', 'back')),
-            ((1, 1, 0), ('bottom', 'right', 'front')),
-            ((0, 1, 0), ('bottom', 'left', 'front')),
-            ((0, 0, 1), ('top', 'left', 'back')),
-            ((1, 0, 1), ('top', 'right', 'back')),
-            ((1, 1, 1), ('top', 'right', 'front')),
-            ((0, 1, 1), ('top', 'left', 'front'))
+        LabeledPoint((0, 0, 0), ('bottom', 'left', 'back')),
```

Comment:
Any reason to explicitly create LPs as opposed to calling `CuboidAnchorCollection.create()`?

Reply:
I had the anchor_test.py existing for my initial anchor implementation. No real reason besides that. I think anchor_test.py should be removed once we're happy with the tests and functionality in test/geometry_test.py. geometry_test.py may be the wrong place for the current tests too